lib/hedwig_mopidy.ex

```diff
@@ -74,4 +74,113 @@ defmodule HedwigMopidy do
   def user(message) do
     if is_map(message.user) do message.user.name else message.user end
   end
-end
+
+  def default_playlist do
+    Application.get_env(:hedwig_mopidy, :default_playlist) || "spotify:user:1241621489:playlist:3XnjVBxdNxNQCMtXR68qMp"
+  end
+
+  def favorites_playlist do
+    Application.get_env(:hedwig_mopidy, :favorites_playlist) || "spotify:user:labzeroinnovations:playlist:00l7ibuNDlOGRvfReyaUge"
+  end
+
+  def rejects_playlist do
+    Application.get_env(:hedwig_mopidy, :rejects_playlist) || "spotify:user:labzeroinnovations:playlist:6TO5iMRRgTGvxEFUWI4D89"
+  end
+
+  defmodule Spotify do
```

Comment:
If we are going to include a module that hooks up to Spotify's Web API it should be in a separate file.
But do we need to define our own Spotify library? There are two others on Hex, with `spotify_ex` having more usage.
https://hexdocs.pm/spotify_ex/Spotify.Playlist.html